docs/migration.md

```diff
@@ -71,6 +75,47 @@ from pydantic.v1.utils import lenient_isinstance
 
 Pydantic V1 documentation is available at [https://docs.pydantic.dev/1.10/](https://docs.pydantic.dev/1.10/).
 
+
+#### Using Pydantic v1 features in a v1/v2 environment
+
+As of `pydantic>=1.10.17` the `pydantic.v1` namespace is available to use within v1,
+meaning that in order to unpin a `pydantic<2` dependency to allow using `v1` features,
+simply replace `pydantic<2` with `pydantic>=1.10.17,<3` and find and replace all:
+
+```python test="skip" lint="skip" upgrade="skip"
+from pydantic.<module> import <object>
+```
+
+to:
+
+```python test="skip" lint="skip" upgrade="skip"
+from pydantic.v1.<module> import <object>
+```
+
+Imports for `pydantic` v1 features for different versions of pydantic.
+
+=== "`pydantic>2,<3`"
+
+    ```python test="skip" lint="skip" upgrade="skip"
+    try:
+        from pydantic.v1.fields import ModelField
+    except ImportError:
+        from pydantic.fields import ModelField
+    ```
+
+=== "`pydantic>=1.10.17,<3`"
```

Comment:
Not sure if we need the `<3` bound...

Reply:
I heard that v3 will get away with the v1 namespace entirely - so was somewhat futureproofing - is this not the case?

Reply:
Ah yes, good point. Good call!